acm/registration/near-edge/overlays/opentelemetry/kustomization.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
+
+namePrefix: otel-edge-collector-
+
+patches:
+- patch: |-
+    - op: replace
+      path: /spec/template/metadata/name
+      value: '{{name}}-openshift-opentelemetry-operator'
+    - op: replace
+      path: /spec/template/spec/source/path
+      value: acm/odh-edge/apps/telemetry
+    - op: replace
+      path: /spec/template/spec/destination/namespace
+      value: ai-edge-collector
```

Shouldn't this be
```suggestion
      value: opendatahub-edge-collector
```
?

---

:+1: